src/Containers/App.js

```diff
@@ -1,124 +1,125 @@
 import { HashRouter, Route } from 'react-router-dom';
 import React, { Component } from 'react';
-import { Row, Col } from 'antd';
+import { Col, Row } from 'antd';
 import 'antd/dist/antd.css';
 import './App.css';
 
 import InlineMenu from '../Components/Menu';
 import Home from '../Components/Home';
 import IssueToken from '../Components/IssueToken';
 
-//Data could be fetched from external source
+// Data could be fetched from external source
 const initialData = [
-  { 
-    key:'1',
-    TokenName: 'TTism',
-    TokenTicker: 'TTT',
-    TotalSupply: 100000,
-    CreationDate: '17 Mai 2019',
-    IssuerName: 'Taurus Group SA',
-    Template:'ERC20',
+  {
+    key: '1',
+    tokenName: 'TTism',
+    tokenTicker: 'TTT',
+    totalSupply: 100000,
+    creationDate: '17 Mai 2019',
```

Comment:
how can i do if i want to render the date with another format, like 17.05.2019 ?